packages/template-express-minimal/package.json

```diff
@@ -1,35 +1,35 @@
 {
-    "name": "template-express-minimal",
-    "version": "2.7.2",
-    "license": "See license in LICENSE",
-    "private": true,
-    "devDependencies": {
-        "pwa-kit-dev": "^2.7.2",
-        "pwa-kit-runtime": "^2.7.2",
-        "supertest": "^4.0.2"
+  "name": "template-express-minimal",
```

Comment:
Are these indentation changes expected in the `package.json` files?

Reply:
Yeah those are mostly formatter changes when I updated eslint config to use 2 space tabs for package.json